stats/test/platform-routes.test.js

```diff
@@ -274,20 +291,37 @@ describe('Platform Routes HTTP request handler', () => {
     })
   })
 
+  beforeEach(async () => {
+    await pgPools.stats.query('DELETE FROM daily_reward_transfers')
+    await pgPools.stats.query('DELETE FROM daily_scheduled_rewards')
+    await pgPools.stats.query('DELETE FROM participants')
+
+    await pgPools.stats.query(`
+      INSERT INTO participants (id, participant_address)
+      VALUES
+        (1, 'to1'),
+        (2, 'to2'),
+        (3, 'to3'),
+        (4, 'address1'),
+        (5, 'address2'),
+        (6, 'address3')
+    `)
```

Comment:
Ditto.

I recommend creating a helper that will update both `participants` and `daily_scheduled_rewards` tables, e.g. `givenDailyScheduledRewards(day, participantAddress, scheduledRewards)`.